docs/spec.yml

```diff
@@ -0,0 +1,673 @@
+openapi: 3.0.0
```

Comment:
Is it worth adding the following to the .gitIgnore file for those of us that use IDEA IDE's?

.idea/**

When I am running this locally, it is prompting me to commit.